shared/chat/conversation/attachment-fullscreen/index.desktop.js

```diff
@@ -56,18 +59,35 @@ class _Fullscreen extends React.Component<Props & OverlayParentProps, State> {
             <Box
               style={collapseStyles([
                 this.props.isZoomed ? styleContentsZoom : styleContentsFit,
-                this.state.loaded ? null : {display: 'none'},
+                this._isLoaded() ? null : {display: 'none'},
               ])}
-              onClick={this.props.onToggleZoom}
+              onClick={() => {
+                if (!this.props.isVideo) {
+                  this.props.onToggleZoom()
+                }
+              }}
+              key={this.props.path}
             >
-              <OrientedImage
-                src={this.props.path}
-                style={this.props.isZoomed ? styleImageZoom : styleImageFit}
-                onLoad={this._setLoaded}
-              />
+              {!this.props.isVideo ? (
+                <OrientedImage
+                  src={this.props.path}
+                  style={this.props.isZoomed ? styleImageZoom : styleImageFit}
+                  onLoad={() => this._setLoaded(this.props.path)}
```

Comment:
this isn't safe, you need to wrap this in a check for _mounted

Reply:
I'm not sure I understand why, what is unsafe here? Isn't this just the same as before but with the path being passed through?

Reply:
so i think we should clean up if we're bothering to touch some of this stuff so, even if its not something you changed per se we likely want to update it. onLoad can be called at any point, even after this thing is unmounted so its never safe to allow that with setState calls